hedgehog/src/Hedgehog/Internal/Report.hs

```diff
@@ -11,9 +11,9 @@ module Hedgehog.Internal.Report (
   , FailureReport(..)
   , FailedInput(..)
 
-  , ShrinkCount
-  , TestCount
-  , DiscardCount
+  , ShrinkCount(..)
+  , TestCount(..)
+  , DiscardCount(..)
```

Comment:
I don't necessarily have a problem exposing the constructors, they're not protecting anything special